src/test/java/uk/gov/companieshouse/api/strikeoffobjections/controller/ObjectionControllerTest.java

```diff
@@ -400,6 +406,85 @@ public void deleteAttachmentWhenUnableToDelete()
         ResponseEntity response = objectionController.deleteAttachment(COMPANY_NUMBER, OBJECTION_ID, ATTACHMENT_ID, REQUEST_ID);
 
         assertEquals(HttpStatus.INTERNAL_SERVER_ERROR, response.getStatusCode());
+    }
+
+    @Test
+    public void testReturnOkStatusForDownload() throws ServiceException {
+        HttpServletResponse httpServletResponse = new MockHttpServletResponse();
+        FileTransferApiClientResponse dummyDownloadResponse = Utils.dummyDownloadResponse();
+        dummyDownloadResponse.setHttpStatus(HttpStatus.OK);
+        when(objectionService.downloadAttachment(REQUEST_ID, OBJECTION_ID, ATTACHMENT_ID, httpServletResponse))
+                .thenReturn(dummyDownloadResponse);
+
+        ResponseEntity responseEntity =
+                objectionController.downloadAttachment(
+                        COMPANY_NUMBER, OBJECTION_ID, ATTACHMENT_ID, REQUEST_ID, httpServletResponse);
+
+        assertEquals(HttpStatus.OK, responseEntity.getStatusCode());
+        assertNull(responseEntity.getBody());
+        assertTrue(responseEntity.getHeaders().isEmpty());
+    }
+
+    @Test
+    public void testReturnUnauthorizedStatusWhenInDownloadResponse() throws ServiceException {
```

Comment:
Could this test and the one below it have better names?
Perhaps testReturnUnauthorizedStatusWhenDownloadResponseUnauthorized

Reply:
I don't really want to repeat the word "Unauthroized" - the name's not that bad surely?

Reply:
To me it wasn't immediately clear what was supposed to be "in" the download response.
My initial thoughts were it was short for invalid response.
Maybe get another opinion on this one, but it's not overly important

Reply:
'testReturnUnauthorizedStatusWhenDownloadFailsAuthorizationChecks' perhaps? (and 'testReturnForbiddenStatusWhenNotAllowedToDownloadAnInfectedFile' maybe)

Reply:
can't we just call it testReturnUnauthorizedStatusForDownload

Reply:
Shorted the name to something clearer